src/server/routes/entity/entity.js

```diff
@@ -166,9 +166,11 @@ module.exports.displayRevisions = (req, res, next, RevisionModel) => {
 
 function _createNote(content, editor, revision, transacting) {
 	if (content) {
+		const revisionId = typeof revision === 'string' ?
```

Comment:
I'd rather we not overload the meaning of this param. It should be possible to do something like the following in `addNoteToRevision()` below:
```javascript
const revision = Revision.forge({id: req.params.id});
_createNote(req.body.note, editorJSON, revision, transacting);
```
Generally, having to check the type of an argument means we're doing something wrong.